apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php

```diff
@@ -271,7 +275,30 @@ function load_block_patterns() {
 
 	Block_Patterns::get_instance();
 }
-add_action( 'init', __NAMESPACE__ . '\load_block_patterns', 20 );
+add_action( 'init', __NAMESPACE__ . '\load_local_block_patterns', 20 );
+
+/**
+ * Load editing toolkit block patterns from the API
+ *
+ * @param obj $current_screen The current screen object.
+ */
+function load_block_patterns_from_api( $current_screen ) {
+	if ( ! apply_filters( 'a8c_enable_block_patterns_api', false ) ) {
+		return;
+	}
+
+	if ( ! function_exists( '\gutenberg_load_block_pattern' ) ) {
```

Comment:
> I couldn't get this to successfully run on an Atomic site.

Like @glendaviesnz I couldn't get it to run on an Atomic (ephemeral) site either. Only when I commented out this check did it work.

`function_exists( '\gutenberg_load_block_pattern' )` returns `false` for me always in my Atomic site.

When I remove this check, the patterns load, albeit containing many invalid blocks 

<img width="419" alt="Screen Shot 2020-10-20 at 3 03 54 pm" src="https://user-images.githubusercontent.com/6458278/96539538-070d7580-12e7-11eb-93db-eaa09c79061f.png">

Reply:
Is the gutenberg plugin loading on the ephemeral sites? That's where this function is.

Reply:
🤕 Sorry for wasting your time @apeatling I forgot to check whether I'd actually enabled the Gutenberg plugin. My bad.
I just assumed that, because I was looking at the Block Editor, it was there. Didn't realize that the function might not be in core.

Reply:
I activated Gutenberg 🤦 and the patterns load. The invalid blocks in the previews all seem to be related to the following: 

> Your site doesn’t include support for the Layout Grid Block block. You can try installing the block, convert it to a Custom HTML block, or remove it entirely.

Installing the Layout Grid Block in the editor once fixes this. 👍

Reply:
Great! No worries on the confusion, good to double check.